base/scene.py

```diff
@@ -22,5 +23,5 @@ def render(self):
     def handle_input(self):
         pass
 
-    def transition_to(self, scene_name):
-        self.scene_manager.transition_to(scene_name)
+    def transition_to(self, scene_name, **kwargs):
```

Comment:
> likely require different variables for future scenes we haven't thought about yet

Typically I don't code for future cases especially if I haven't thought them through. We need to be careful here cause this can introduce bugs to someone trying to interact with the framework. `kwargs` in the method signature doesn't help to tell what the method does.  

> Basically we need to be able to pass any information in the scene's creation/render 

No we don't, a scene should be passed a context object from the previous scene, `kwargs` is not a suitable vehicle for this as we have control of the previous scene and should know what exactly to feed the next scene. 

> without having to add every args on every sceness

Agree, If we get here we have a bigger problem and need to step back and rethink things.